index.bs

```diff
@@ -1648,6 +1829,12 @@ Note: Request and watch IDs are not unique across agents.  An agent can combine
 a request ID with a unique identifier for the agent that sent it (like its
 certificate fingerprint) to track requests across multiple agents.
 
+Extension fields should use string keys to avoid conflicts with integer keys in
```

Comment:
I think this was fallout from the merging/rebasing.  It should go at the end of Protocol Extensions.  I can fix this in a commit.

Reply:
Fixed